Twitch/Twitch.Base/Services/NewAPI/WebhooksService.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Twitch.Base.Models.NewAPI.Webhooks;
+
+namespace Twitch.Base.Services.NewAPI
+{
+    /// <summary>
+    /// The APIs for Webhook-based services.
+    /// </summary>
+    public class WebhooksService : NewTwitchAPIServiceBase
+    {
+
+        /// <summary>
+        /// Creates an instance of the WebhooksService
+        /// </summary>
+        /// <param name="connection"></param>
+        public WebhooksService(TwitchConnection connection) : base(connection) { }
+
+        /// <summary>
+        /// Gets the Webhook subscriptions of a user identified by a Bearer token, in order of expiration.  Requires App Token.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Missing return comment

Reply:
Here and in other methods in the class, not gonna leave a comment for each one :)

Reply:
Completed